scripts/yaml_check.py

```diff
@@ -0,0 +1,41 @@
+import yaml
+import sys
+
+def tiered_extract(db):
+  out = []
+  for _, entry in db.items():
+    for _, values in entry.items():
+      for name, decl in values.items():
+        if decl is not None and decl != '' and '/' not in decl:
+          out.append((name, decl))
+  return out
+
+def print_list(fn, pairs):
+  with open(fn, 'w') as out:
+    for (id, val) in pairs:
+      out.write(f'{id}\n{val.strip()}\n\n')
+
+hundred_yaml = sys.argv[1]
+overview_yaml = sys.argv[2]
+undergrad_yaml = sys.argv[3]
+
+with open(hundred_yaml, 'r') as hy:
+  hundred = yaml.safe_load(hy)
+with open(overview_yaml, 'r') as hy:
+  overview = yaml.safe_load(hy)
+with open(undergrad_yaml, 'r') as hy:
+  undergrad = yaml.safe_load(hy)
+
+hundred_decls = []
```

Comment:
```suggestion
hundred_decls: List[Tuple[str, str]] = []
```
Let's help readers (and `mymy` is we want).